docs/branding.md

```diff
@@ -19,3 +19,20 @@ You can change the logo shown on the CMS login screen using the `login-logo` con
 	}
 }
 ```
+
+## Custom admin colour scheme
+
+To register a custom admin colour scheme add the following function on `admin_init` action.
```

Comment:
Let's expand on this to say why you'd want to do it - in our case we're recommending it for accessibility purposes.
```suggestion
If the default colour scheme does not meet the accessibility needs of your staff you can register custom schemes to provide high contrast alternatives. To register a custom admin colour scheme add the following function on `admin_init` action:
```